src/libcollectionstest/slice.rs

```diff
@@ -444,7 +444,7 @@ fn test_sort() {
     let mut v: [i32; 0] = [];
     v.sort();
 
-    let mut v = [0xDEADBEEFu64];
+    let mut v = [0xDEADBEEF];
```

Comment:
doesn't this give a literal overflow warning because the value is too big for an `i32`?